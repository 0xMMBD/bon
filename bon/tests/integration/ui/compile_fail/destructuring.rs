#[bon::builder]
fn destructuring((x, y): (u32, u32)) {
    let _ = x;
    let _ = y;
}

fn main() {}
