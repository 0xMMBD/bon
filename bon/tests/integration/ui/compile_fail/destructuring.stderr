error: Only simple identifiers in function arguments are supported. Parameter names influence the setter method names. If you need to destructure a function parameter, then do it inside of the function body.
 --> tests/integration/ui/compile_fail/destructuring.rs:2:18
  |
2 | fn destructuring((x, y): (u32, u32)) {
  |                  ^^^^^^
